plugins/catalog-backend-module-github/config.d.ts

```diff
@@ -16,6 +16,70 @@
 
 import { TaskScheduleDefinitionConfig } from '@backstage/backend-tasks';
 
+interface GithubConfig {
```

Comment:
Sorry, don't want to separate this out into a common type as the namespace is global 😅, will need to keep it as it was